src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Rectangle extends Properties {
+    private final int width = 6;
+    private final int length = 8;

```

Comment:
```suggestion
    private int width;
    private int length;

```
You don't have to provide this data yourself. Create a constructor where you take them and color. Fix in all places!